bundles/com.espressif.idf.terminal.connector.serial/src/com/espressif/idf/terminal/connector/serial/launcher/GDBStubDebuggerLauncher.java

```diff
@@ -230,15 +241,16 @@ private void createXMLConfig() throws Exception {
 
 		Transformer tr = TransformerFactory.newInstance().newTransformer();
 		tr.setOutputProperty(OutputKeys.INDENT, "yes"); //$NON-NLS-1$
-		String launchFile = project.getLocation().makeAbsolute().toString().concat("/") // $NON-NLS-1$
+		String launchFile = project.getLocation().makeAbsolute().toString().concat("/") //$NON-NLS-1$
 				.concat(GDBSTUB_DEBUG_LAUNCH_CONFIG_FILE);
 		tr.transform(new DOMSource(dom), new StreamResult(new File(launchFile)));
 		project.refreshLocal(IResource.DEPTH_INFINITE, new NullProgressMonitor());
 	}
 
-	@SuppressWarnings("restriction")
-	private class GDBLaunchConfig extends LaunchConfiguration {
-		protected GDBLaunchConfig(IFile file) {
+	private class GDBLaunchConfig extends LaunchConfiguration
+	{
+		protected GDBLaunchConfig(IFile file)
+		{
 			super(file);
 		}
 	}
```

Comment:
Unless needed I would suggest using `LaunchManager.getLaunchConfiguration(file)` for creating a launch configuration